shell/platform/android/io/flutter/plugin/common/StandardMessageCodec.java

```diff
@@ -208,10 +208,8 @@ protected static final void writeAlignment(ByteArrayOutputStream stream, int ali
   protected void writeValue(ByteArrayOutputStream stream, Object value) {
     if (value == null || value.equals(null)) {
       stream.write(NULL);
-    } else if (value == Boolean.TRUE) {
```

Comment:
I don't have a strong opinion. Boolean.TRUE.equals(value) might be easier to read than the ternary?